po/es.po

```diff
@@ -141,92 +114,83 @@ msgstr "Configure una nueva habilidad"
 
 #: lib/dialogue-agent/execution_dialogue_agent.ts:251
 msgid "No contact matches “${name}”."
-msgstr "Más de un contacto coincide con “${name}”. ¿A quién se refiere?"
+msgstr "No contactos coincide con “${name}”."
 
 #: lib/dialogue-agent/execution_dialogue_agent.ts:257
 msgid "Who do you want to contact?"
-msgstr "¿Con quién desea contactar?"
+msgstr "¿Quién quieres contactar?"
 
 #: lib/dialogue-agent/execution_dialogue_agent.ts:338
 msgid "Sorry, I cannot find any ${entity_type} matching “${name}”."
-msgstr "Lo siento, pero no encuentro ningún ${entity_type} que coincida con ${name}”."
+msgstr "Lo siento, no puedo encontrar ningún ${entity_type} que coincida con ${name}”."
 
 #: lib/dialogue-agent/execution_dialogue_agent.ts:378
 msgid "Sorry, I cannot find any location matching “${location}”."
-msgstr ""
-"Lo siento, pero no puedo encontrar ninguna ubicación que coincida con “${location}”."
+msgstr "Lo siento, no puedo encontrar ninguna ubicación que coincida con “${location}”."
 
 #: lib/dialogue-agent/execution_dialogue_agent.ts:413
 #, fuzzy
 msgid "What is your phone number?"
-msgstr "¿Cuál es la dirección de su domicilio?"
+msgstr "¿Cuál es su número de teléfono?"
```

Comment:
After editing a translation with the fuzzy marker, remove it, or the translation won't be used.